pkg/cluster/internal/kubeadm/config.go

```diff
@@ -173,7 +183,7 @@ kubernetesVersion: {{.KubernetesVersion}}
 clusterName: "{{.ClusterName}}"
 {{ if .KubeadmFeatureGates}}featureGates:
 {{ range $key, $value := .KubeadmFeatureGates }}
-  "{{ $key }}": {{ $value }}
+  "{{ (StructuralData $key) }}": {{ $value }}
```

Comment:
StructuralData is a special function to tell safetext that this is *supposed* to result in a dynamic input-controlled key and that's expected.